vignettes/getting-started.Rmd

```diff
@@ -0,0 +1,156 @@
+if (fs::file_exists("../inst/nonmem/2.yaml")) fs::file_delete("../inst/nonmem/2.yaml")
+if (fs::file_exists("../inst/nonmem/3.yaml")) fs::file_delete("../inst/nonmem/3.yaml")
+if (fs::file_exists("../inst/nonmem/2.ctl")) fs::file_delete("../inst/nonmem/2.ctl")
+if (fs::file_exists("../inst/nonmem/3.ctl")) fs::file_delete("../inst/nonmem/3.ctl")
+if (fs::file_exists("../inst/nonmem/babylon.yaml")) fs::file_delete("../inst/nonmem/babylon.yaml")
+```
+
+# Introduction
+
+`rbabylon` is an R interface for running `babylon` which, according to [its README](https://github.com/metrumresearchgroup/babylon/), "is (will be) a complete solution for managing projects involving modeling and simulation with a number of software solutions used in pharmaceutical sciences." Currently, only NONMEM is supported, so this vignette will only address that. 
+
+This "Getting Started with rbabylon" vignette takes the user through some basic scenarios for modeling with NONMEM using `rbabylon`, introducing you to its basic workflow and functionality. 
+
+# Setup
+
+There is some initial setup to get `rbabylon` configured. First, load the library and make sure you have installed `babylon` (often aliased as `bbi`) and told `rbabylon` where to find it. **POINT TO BBI INSTALLATION INSTRUCTIONS, do these exist?**
+
+```{r bbi_exe_path}
+library(rbabylon)
+options('rbabylon.bbi_exe_path' = '/data/apps/bbi')
+```
+
+Next, initialize `babylon` by pointing it to a working installation of NONMEM, and default version to use.
+
+```{r setup, eval=FALSE}
+bbi_init(.dir = "../inst/nonmem",     # the modeling directory, where your control streams will be
+         .nonmem_dir = "/opt/NONMEM", # location of NONMEM installation
+         .nonmem_version = "nm74gf")  # default NONMEM version to use
+```
+
+Finally, it is best to set the modeling directory. By doing this, you do not need to pass the full path to model files, but only the path relative to the specified modeling directory. Pass a path relative to your current working directory to `set_model_directory()`
+
+```{r set_model_directory, echo = FALSE}
+set_model_directory("../inst/nonmem")
+```
+
+# Initial Modeling Run
+
+## Create Model Object
+To begin modeling, first create a model object. This is an S3 object with an accompanying YAML file, kept in your modeling directory. The YAML serves as the "record of truth" for the model, and will persist changes you make to the model as you go.
+
+The `.yaml_path` argument should have the same name (but with a `.yaml` extension) as your base model control stream. For instance, the command below assumes you have a control stream name `1.ctl` or `1.mod` in the model directory you just defined. The `.description` argument is required and corresponds to what would be in the `$PROB` definition in a control stream.
+
+```{r first model}
+mod1 <- new_model(.yaml_path = "1.yaml", .description = "our first model")
```

Comment:
this is not propagating `model_path` currently, as such I can't get past this point when I run through it